README.md

```diff
@@ -45,24 +45,27 @@ To call an endpoint you must create a `Client` object.
 
 ```go
 import (
+    "net/http"
     "os"
 
     "github.com/plaid/plaid-go/plaid"
 )
 
-client := plaid.NewClient(
+clientOptions := plaid.ClientOptions{
     os.Getenv("PLAID_CLIENT_ID"),
     os.Getenv("PLAID_SECRET"),
     os.Getenv("PLAID_PUBLIC_KEY"),
     plaid.Sandbox, // Available environments are Sandbox, Development, and Production
-)
+    &http.Client{},
```

Comment:
nit: it might be nice to say here that `http.Client{}` is optional.

Reply:
⚡

Reply:
⚡